src/content/plugins/profiling-plugin.md

```diff
@@ -27,8 +27,9 @@ new webpack.debug.ProfilingPlugin({
 
 In order to view the profile file:
 
-- Run webpack with `ProfilingPlugin`.
-- Go to Chrome, open the `Profile Tab`.
-- Drag and drop generated file (`events.json` by default) into the profiler.
+1. Run webpack with `ProfilingPlugin`.
+1. Go to Chrome and open devtools.
+1. In Chrome devtools, go to the the `Performance` tab (formerly `Timeline`).
```

Comment:
I guess this one and the one above can be combined? Other than that looks good 👍🏽